linuxNew/Jenkinsfile

```diff
@@ -65,6 +139,18 @@ def buildAndTest() {
     archiveArtifacts artifacts: '**/build/ospackage/*,**/build/reports/**,**/packageTest/dependencies/deb/*'
 }
 
+def buildNoTest() {
+    // Install Adoptium GPG key for RPM signing
+    withCredentials([file(credentialsId: 'adoptium-artifactory-gpg-key', variable: 'GPG_KEY')]) {
+        if (DISTRO != "Debian") {
+            sh("./gradlew packageJdk${DISTRO} --parallel -PPRODUCT=${PRODUCT} -PPRODUCT_VERSION=${VERSION} -PGPG_KEY=${GPG_KEY} || true")
+        } else {
+            sh("./gradlew packageJdk${DISTRO} --parallel -PPRODUCT=${PRODUCT} -PPRODUCT_VERSION=${VERSION} || true")
```

Comment:
Why the `|| true`? Is there a reason why we'd want to ignore build failures?

Reply:
@sophia added it [in another PR](https://github.com/adoptium/installer/pull/374) although it looks like it was [subsequently removed again](https://github.com/adoptium/installer/pull/383/files) after I started this PR, so I'll look at removing it from mine.

Reply:
By the look of it we are stil getting failures when that is added back in (even without my PR - it looks like @sophia-guo 's job (without this PR) is getting some failures at the moment now that the `|| true` has been removed.

Reply:
Yes, I know. Rather than re-introducing `| true` it we should look at fixing the failures. Is the failure you are talking about this one? https://github.com/adoptium/installer/issues/385